content/preferences-settings/general.md

```diff
@@ -8,7 +8,7 @@ draft: false
 Control the overall look and feel of darktable.
 
 interface language
-: Set the language of the user interface. The system default is marked with an * (needs a restart)
+: Set the language of the user interface. It includes the option to enable sentence case (`en@truecase`) for `en` (currently the `US` variant, and may not apply to subsequent variants). The system default is marked with an * (needs a restart)
```

Comment:
I probably wouldn't bother mentioning "variants" of EN at the moment since there's currently only one

Reply:
@elstoc, I agree. BTW, do you know why I don't see a way to easily incorporate this suggestion, like https://github.com/github/docs/blob/e3a169c100d6542bd8178dd47590f0047739107b/content/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request.md#applying-suggested-changes explains?

Reply:
@elstoc, is https://github.com/darktable-org/dtdocs/pull/677/commits/07d2014f69b706c9dca85cdcd70e846598d08f50#diff-6c3d86dab3e3c7b47a97eafe238a53f8dbddc3845634725ed8e08221c9533f0aR11 adequate?